### PR TITLE
test

### PR DESCRIPTION
**Pull request by Benoit1108**

### CODE REVIEW

File: exampleSite/data/fr/about.yml [hunk 1]
```diff
@@ -7,9 +7,8 @@ about:
     title: Nous sommes créatif
     content: Ceci est avant tout un test
   - icon: ti-cup
-    title: We're Professional
-    content: Lorem ipsum dolor sit amet, consectetur adipisicing elit. Velit, nihil,
-      libero, perspiciatis eos provident laborum eum dignissimos.
+    title: Nous sommes professionnels
```
Comment (divinerites) [inline exampleSite/data/fr/about.yml:10]: Please can you avoid to push your own tests to the public official repository ? Thanks.

**Review by divinerites — CHANGES_REQUESTED** [review]
Please can you avoid to push your own tests to the public official repository ? Thanks.

**Comment by divinerites** [comment]
Can you STOP spamming the official theme ? Thanks.

File: exampleSite/data/fr/about.yml (hunk 1 above)
Comment (mehedi-sharif) [reply exampleSite/data/fr/about.yml:10]: I made a report on his profile . Is there any way to block him or something ? @divinerites

Comment (Codebreaker101) [reply exampleSite/data/fr/about.yml:10]: @mehedi-sharif https://docs.github.com/en/free-pro-team@latest/github/building-a-strong-community/blocking-a-user-from-your-organization

**Pull request closed without merging**